### ISSUE TITLE
Latest version does not compile on IOS

### ISSUE THREAD

**Issue by tkuben** [BranchMetrics/Cordova-Ionic-PhoneGap-Deferred-Deep-Linking-SDK#63]
Hi, 

I pulled the latest version of the SDK and when I try to build for ios, I get the following error. Any thoughts?

```
Undefined symbols for architecture arm64:

  "_OBJC_CLASS_$_CSSearchableItem", referenced from:

      objc-class-ref in BranchCSSearchableItemAttributeSet.o

  "_OBJC_METACLASS_$_CSSearchableItemAttributeSet", referenced from:

      _OBJC_METACLASS_$_BranchCSSearchableItemAttributeSet in BranchCSSearchableItemAttributeSet.o

  "_OBJC_CLASS_$_CSSearchableIndex", referenced from:

      objc-class-ref in BranchCSSearchableItemAttributeSet.o

  "_OBJC_CLASS_$_CSSearchableItemAttributeSet", referenced from:

      _OBJC_CLASS_$_BranchCSSearchableItemAttributeSet in BranchCSSearchableItemAttributeSet.o

ld: symbol(s) not found for architecture arm64

clang: error: linker command failed with exit code 1 (use -v to see invocation)


```

Cheers, 
TJ

**Comment by aaustin** [comment]
Hey @tkuben - we require the CoreSpotlight.framework and you're getting a build error because it's not included. I can have the engineers look at whether it's possible to automatically include the framework at import time.

**Comment by tkuben** [comment]
@aaustin. Thank you. That allowed me to build the latest version of the SDK. However, the issue: https://github.com/BranchMetrics/Cordova-Ionic-PhoneGap-Deferred-Deep-Linking-SDK/issues/62 still is a critical blocker for us even with the latest code. 

Cheers, 
TJ

**Comment by aaustin** [comment]
I understand - Working on it.

**Comment by elliotstokes** [comment]
I think I may have solved this in #250.
